crates/core/src/contracts/account.rs

```diff
@@ -43,7 +43,12 @@ pub trait Account<'a, P: Provider + Send + Sync + 'a> {
             _ => Err(EthApiError::from(err)),
         })?;
 
-        Ok(vec_felt_to_bytes(bytecode))
+        if bytecode.is_empty() {
+            return Ok(Bytes::default());
```

Comment:
Did you check what Bytes::default() is? iirc it's not 0

Reply:
`Bytes::default()` is `0x` so empty, should be ok